vpr/src/route/rr_graph.cpp

```diff
@@ -310,6 +318,8 @@ static void build_rr_graph(const t_graph_type graph_type,
                            int* wire_to_rr_ipin_switch,
                            int* Warnings);
 
+std::vector<t_pin_loc> get_pin_loc(const e_pin_type pin_type, const t_physical_tile_type_ptr Type);
```

Comment:
Forgot to remove this. It was used during my testing and is not necessary, so I will fix this in the next commit.